environment.yml

```diff
@@ -0,0 +1,94 @@
+name: moran
```

Comment:
I suggest we do, when we get the paper out there we'll basically be saying: if you use this exact environment (so for example the jupyter notebooks are happy with the env name etc) and run the code then you will get the same results we do.